docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md

```diff
@@ -15,39 +15,30 @@ Hardware requirements:
 <a name="on_Windows"></a>
```

Comment:
In the previous paragraph, please make the following edits. 
replace Speed up by speed-up
and speeds up by speed-ups